cpu/kinetis/include/cpu_conf_kinetis.h

```diff
@@ -128,6 +128,16 @@ extern "C"
 #endif /* KINETIS_RNGA */
 /** @} */
 
+/**
+ * @name Timer Control Register missing definitions for MK20D7.h
+ * @{
+ */
+#if !defined(PIT_TCTRL_CHN_MASK)
+#define PIT_TCTRL_CHN_MASK                       0x4u
```

Comment:
maybe wrap the defines with parenthesis

Reply:
I wanted to keep [the original vendor file style](https://github.com/astralien3000/RIOT/blob/teensy31/cpu/kinetis/include/vendor/MK20D7.h#L5361), should I change it anyway ?

Reply:
Since it's now in an original RIOT source file, it should be compliant with RIOT coding conventions, so I would answer yes.